backend/database/migrations/1730574250954.sql

```diff
@@ -1,5 +1,7 @@
--- Basic profiles, courses and mappings
+-- Basic profiles, courses for issue #79
```

Comment:
could you remove the tag for the issue (you can keep it so that it says Basic profiles and courses